### PR TITLE
removed @uimaxbai's grabify link and some other small tweaks

### PR DESCRIPTION
**Pull request by Yeetface513**
Hope you accept these qol and phrasing fixes

### CODE REVIEW

**Comment by Yeetface513** [comment]
also removed the grabify link from @uimaxbai @rocketman12435 please merge this

**Comment by Yeetface513** [comment]
do it

**Review by Yeetface513 — COMMENTED** [review]
an urgent fix

**Comment by Yeetface513** [comment]
must be done with haste

**Comment by uimaxbai** [comment]
I did not add any of such link. The commit history clearly states that @rocketman12435 did it.

**Comment by uimaxbai** [comment]
Also there is inappropriate language in the SECURITY.md file, I would recommend not merging this.

**Comment by uimaxbai** [comment]
nvm its been changed

**Comment by binimum** [comment]
It also puts my name in strikethrough in the index.html file (very inappropriate, do not accept).

**Comment by binimum** [comment]
I did not place any link of such, the commit history clearly states that @rocketman12435 placed it in.

File: index.html [hunk 1]
```diff
@@ -155,10 +155,10 @@ <h3 id="delaydiv" class="carahead"><b>Lost items found yesterday:</b></h3>
     </div>
 
     <br />
-
-      <footer class="footer-text">Idea creation, website design and JavaScript by Akshaj Sharma (@Rocketman12435 on GitHub)</footer></br>
-      <footer class="footer-text">Styling improvements by @thisiscoding1234 on GitHub</footer></br>
-      <footer class="footer-text">Other contributors: @werdl, @Neodel, @2010person, @Yeetface513, @uimaxbai (all on GitHub)</footer>
+      <p>Main contributors: @werdl, @Neodel, @2010person, @Yeetface513 and <s>@uimaxbai</s> (all on GitHub)</p>
```
Comment (thisiscoding1234) [inline index.html:158]: The strikethrough is entirely inappropriate.

File: 404.html
Comment (thisiscoding1234) [inline 404.html:1]: This looks fine and adds information

File: README.md
Comment (thisiscoding1234) [inline README.md:1]: This looks good.

File: SECURITY.md
Comment (thisiscoding1234) [inline SECURITY.md:1]: This also LGTM.

File: index.html [hunk 2]
```diff
@@ -60,7 +60,7 @@
 
     <div class="w3-container w3-center w3-animate-right" id="ani15">
       <h1 id="header">
-        <em><b>Have you lost something?</b></em>
+        <em><b>Have you lost an item?</b></em>
```
Comment (thisiscoding1234) [inline index.html:63]: This re-wording is better.

File: index.html (hunk 1 above)
Comment (thisiscoding1234) [reply index.html:158]: Please remove.

File: login/index.html
Comment (thisiscoding1234) [inline login/index.html:1]: Looks fine.

**Review by thisiscoding1234 — CHANGES_REQUESTED** [review]
Please make a change to the index to remove the strikethrough on @uimaxbai's name. Otherwise, it LGTM.

**Comment by rocketman12435** [comment]
> I did not place any link of such, the commit history clearly states that @rocketman12435 placed it in.

Actually Max I didn't change anything, I just added all the files that were in the previous repo to a new repo

**Comment by rocketman12435** [comment]
Anything you have put on the files, be it helpful or not, has been put into the repo, so you must tell me where it is to remove it

**Comment by Yeetface513** [comment]
it's on the login page its the long link I removed (its a grabify link click it to find out)

**Pull request merged**